<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SIDCD495" name="com.xp8.test.XP8_Safeguard_Test" tests="18" failures="0" timestamp="24 Nov 2017 09:17:38 GMT" time="1519.501" errors="2">
  <testcase name="Safeguard_001_Presence_Of_SG_In_Scout" time="53.272" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_013_Verify_application_Pin_TimeOut" time="77.482" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_018_Check_Invalid_Pin_Verification" time="87.233" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_019_Pin_Acccepts_Only_Num_Verification" time="77.767" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_005_verify_Default_States_Of_All_Options_SG" time="56.733" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_004_Navigate_To_Help_Fetch_Pin" time="42.115" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_010_emergency_Call_Check" time="58.925" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_017_Change_Pin_Verification" time="206.404" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_016_max_Limit_Code_Verification" time="31.157" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_011_select_All_In_Feature_Check" time="151.599" classname="com.xp8.test.XP8_Safeguard_Test">
    <error type="java.lang.AssertionError" message="The following asserts failed:
null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at com.xp8.util.XP8_Safeguard_Util.launchFeatures(XP8_Safeguard_Util.java:1098)
	at com.xp8.util.XP8_Safeguard_Util.check_Features_Tab(XP8_Safeguard_Util.java:872)
	at com.xp8.test.XP8_Safeguard_Test.Safeguard_011_select_All_In_Feature_Check(XP8_Safeguard_Test.java:247)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- Safeguard_011_select_All_In_Feature_Check -->
  <testcase name="Safeguard_008_verify_Unselect_All_Apps_And_Check" time="81.488" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_015_min_Code_Verification" time="83.794" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_012_Unselect_All_Features_And_Check" time="157.076" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_014_Verify_App_Pin_Time_Out_One_Min" time="74.946" classname="com.xp8.test.XP8_Safeguard_Test">
    <error type="java.lang.AssertionError" message="The following asserts failed:
null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at com.xp8.util.XP8_Safeguard_Util.pin_Time_Out_Set_To_1_Min(XP8_Safeguard_Util.java:1691)
	at com.xp8.test.XP8_Safeguard_Test.Safeguard_014_Verify_App_Pin_Time_Out_One_Min(XP8_Safeguard_Test.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- Safeguard_014_Verify_App_Pin_Time_Out_One_Min -->
  <testcase name="Safeguard_002_Default_State_SG_In_Scout" time="37.972" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_009_select_few_And_Check" time="54.267" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_003_Verify_Ver_Help_Default_State_SG_Scout" time="38.903" classname="com.xp8.test.XP8_Safeguard_Test"/>
  <testcase name="Safeguard_007_verify_Default_State_Of_All_Apps_In_App_Option" time="148.368" classname="com.xp8.test.XP8_Safeguard_Test"/>
</testsuite> <!-- com.xp8.test.XP8_Safeguard_Test -->
